order/buyer/issue.md

```diff
@@ -0,0 +1,133 @@
+
+{% hint style="danger" %}
+`item.number` should be unique within your company and never change.
+Never renumber or re-use `item.number`s.
+{% endhint %}
+
+#### Requested planned delivery schedule
+
+- `line.deliverySchedule`: the requested planned delivery schedule. Provide at least one or multiple delivery schedule lines.
+- `deliverySchedule.position`: the position in the delivery schedule. Not to be confused with the `line.position`
+- `deliverySchedule.date`: the requested delivery date of this delivery schedule position. Date has ISO 8601 date `yyyy-MM-dd` format
+- `deliverySchedule.quantity`: the requested quantity of this delivery schedule position. Quantity has a decimal `1234.56` format with any number of digits.
+
+{% hint style="danger" %}
+`deliverySchedule.position` should be unique within the delivery schedule and never change.
+Never renumber or re-use `deliverySchedule.position`s.
+{% endhint %}
+
+#### Requested prices
+
+- `lines.prices`: the requested price. Advised is to provide only `netPrice` for its simplicity, used by most buyers, or alternatively `grossPrice` together with `discountPercentage`. 
+- `priceInLocalCurrency`: at least provide a price in the local currency of the supplier, like `CNY` in China.
+- `priceInBaseCurrency`: if available provide a price in your base currency, like `EUR` in the EU.
+- `value`: the price value has a decimal `1234.56` format with any number of digits.
+- `currencyIso`: the 3-letter currency code according to ISO 4217, like `EUR`, `USD` and `CNY`
+- `priceUnitOfMeasureIso`: the 3-letter price unit according to ISO 80000-1. The purchase unit and price unit may be different.
+- `priceUnitQuantity`: the item quantity at which the price applies. Typically this is 1 (unit price) or 100 (the price applies to 100 items)
+
+#### Secondary line fields
+
+- `description`: a free format additional description of this line
+- `terms`: the line terms as agreed with your supplier
+- `terms.contractNumber`: the agreed framework contract number
+- `terms.contractPosition`: the related position within the framework contract
+- `projectNumber`: Your project number reference
+- `productionNumber`:  Your production number reference
+- `salesOrderNumber`:  Your sales order reference (not be confused with the supplier sales order number)
+- `indicators`: see [Indicators](indicators.md)
+- `properties`: are key-value based custom fields. You can use as many as needed, but too many will clutter the portal.  Use `\n` for a new line in the value.
+- `notes`: are simple custom fields.You can use as many as needed, but too many will clutter the portal. Use `\n` for a new line.
+- `documents`: contain meta data and link of attached documents. See [Attach a document to a line](attach-document.md)
+
+#### New order meta data
+
+- `erpIssueDateTime`: Date and time the order was originally issued in your ERP system. `DateTime` has ISO 8601 local date/time format `yyyy-MM-ddThh:mm:ss`
```

Comment:
Link to standarts page?

Reply:
Again redundant info to prevent unnecessary support requests. Goal is make API development self service as we do not have people for api support for 50-100 customers

Reply:
But I added the link